cvat/apps/engine/prepare.py

```diff
@@ -177,6 +195,14 @@ def frame_sizes(self):
         container.seek(offset=next(iter(self.key_frames.values())), stream=video_stream)
         for packet in container.demux(video_stream):
             for frame in packet.decode():
+                if video_stream.metadata.get('rotate'):
+                    frame = av.VideoFrame().from_ndarray(
+                        rotate_image(
+                            frame.to_ndarray(format='bgr24'),
+                            360 - int(container.streams.video[0].metadata.get('rotate'))
```

Comment:
I have a couple of questions here:
- what the method returns: video_stream.metadata.get('rotate')?
- what is the difference between previous expression and container.streams.video[0].metadata.get('rotate')
- can container.streams.video[0].metadata.get('rotate') be always converted to int?

Reply:
> what the method returns: video_stream.metadata.get('rotate')?

string of rotation angle

> what is the difference between previous expression and container.streams.video[0].metadata.get('rotate')

this is analogue

> can container.streams.video[0].metadata.get('rotate') be always converted to int?

I checked for some values like 60, 90, 180, 270.. and in such cases - yes